### PR TITLE
Features/22

### PR DESCRIPTION
**Pull request by manuhortet**
v2.0.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,18 +62,23 @@ Host: <host ip> # default: 0.0.0.0
 Port: <host port> # default: 8089
 
 [Microsoft Teams]
-Connector: <webhook url> # required value
+connector1: <webhook url> # At least one connector is required
```
Comment (dortegau) [inline README.md]: Why that keys are now in lowercase? :wink:

File: README.md [hunk 2]
```diff
@@ -62,18 +62,23 @@ Host: <host ip> # default: 0.0.0.0
 Port: <host port> # default: 8089
 
 [Microsoft Teams]
```
Comment (dortegau) [inline README.md]: IMHO you should specify that keys not followed by an integer number are still supported

File: requirements.txt [hunk 3]
```diff
@@ -1,3 +1,5 @@
 ## Requirements
```
Comment (jmonterrubio) [inline requirements.txt]: I prefer to set versions in the requirements files. Maybe you could add the version to all of them ;)

File: prom2teams/message/parser.py [hunk 4]
```diff
@@ -27,15 +26,12 @@ def check_fields(json_alerts_attr, json_alerts_labels_attr, json_alerts_annotati
                 alert_fields[alert_field_key] = 'unknown'
     return alert_fields
 
-def parse(json_str):
```
Comment (jmonterrubio) [inline prom2teams/message/parser.py]: Now that you are using flask-restplus, it's easier to user http://flask-restplus.readthedocs.io/en/stable/parsing.html
Also is a win to use http://flask-restplus.readthedocs.io/en/stable/swagger.html

File: tests/data/jsons/all_ok.json [hunk 5]
```diff
@@ -8,13 +8,13 @@
         "alertname": "DiskSpace",
         "device": "rootfs",
         "fstype": "rootfs",
-        "instance": "cs30.evilcorp",
```
Comment (jnogol) [inline tests/data/jsons/all_ok.json]: It's only an aesthetic thing, but I would either change L18 to `idealista.Test` as well (and all the instances and summaries in all the json files) or switch back to the previous (and geekier :P ) value. Other than that, really nice job ;)

Comment (manuhortet) [reply tests/data/jsons/all_ok.json]: Switched back to the MR R reference :smile:

File: requirements.txt (hunk 3 above)
Comment (manuhortet) [reply requirements.txt]: Done, I will take that into account from now on :+1:

File: prom2teams/message/parser.py [hunk 6]
```diff
@@ -27,15 +28,12 @@ def check_fields(json_alerts_attr, json_alerts_labels_attr, json_alerts_annotati
...
+def parse(json_values):
     parsed_alarms = {}
-
     for i, alert in enumerate(json_values['alerts']):
         json_alerts_attr = alert
         json_alerts_labels_attr = json_alerts_attr['labels']
         json_alerts_annotations_attr = json_alerts_attr['annotations']
-        parsed_alarms['alarm_' + str(i)]=check_fields(json_alerts_attr, json_alerts_labels_attr, json_alerts_annotations_attr)
-
+        parsed_alarms['alarm_' + str(i)] = check_fields(json_alerts_attr, json_alerts_labels_attr, json_alerts_annotations_attr)
```
Comment (juanriaza) [inline prom2teams/message/parser.py]: `f'alarm_{i}'`

File: README.md (hunk 1 above)
Comment (manuhortet) [reply README.md]: Switched back to the initial capital letter :)

File: README.md (hunk 2 above)
Comment (manuhortet) [reply README.md]: Changed variable names to "Connector1" and "ConnectorTwo", I guess it seems clear now :)

**Pull request merged**